dist/pact-consumer-js-dsl.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 var Pact = Pact || {};
+Pact.IsNodeJs = typeof module === 'object' && typeof module.exports === 'object';
+
+if (Pact.IsNodeJs) {
+  module.exports = Pact;
+}
 
```

Comment:
That PR is based upon the node module `xmlhttprequest` which we [discussed](https://github.com/DiUS/pact-consumer-js-dsl/issues/15#issuecomment-67925346) a while back.

I think @markdalgleish has a nicer solution for making the differences between node and the browser more transparent using browserify, which gives us a dependency management solution as well.

But to be honest I'd like to focus on getting this change through before going through a refactoring exercise to clean up the nodejs checks. The length of time it's taking to get this PR approved is hurting my ability to get Pact widely adopted at my company.

If I make the changes discussed above are there any other concerns which will prevent this getting approved?